src/js/utils/parser.js

```diff
@@ -142,9 +142,9 @@ define([
      * Determine the adaptive type
      */
     parser.adaptiveType = function(duration) {
-        if (duration !== -1) {
+        if (duration !== 0) {
```

Comment:
You may know this but, before this change, a duration of 0 returned undefined, now it returns "vod"


| Current | Value |
|---------|---|
| [-Infinity, -120]  | DVR |
| (-120, 0) | LIVE |
| [0, Infinity) | VOD |
| [Infinity, Infinity] | LIVE |

What do you think about this?

| Suggested | Value |
|---------|---|
| [-Infinity, 0)  | DVR |
| [0, 0] | Unknown |
| (0, Infinity) | VOD |
| [Infinity, Infinity] | LIVE |